enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/identity/MemberId.java

```diff
@@ -38,7 +38,7 @@ public MemberId( UUID uuid )
     {
         Objects.requireNonNull( uuid );
         this.uuid = uuid;
-        shortName = uuid.toString().substring( 0, 8 );
+        shortName = uuid.toString();//.substring( 0, 8 );
```

Comment:
That substring makes small value UUIDs come out as eight 0's. But it is the way it used to be so reverted.